File: Wisdom-Knowledge/afrodite/afrodite.py

class Afrodite:
    def __init__(self):
        self.name = "Afrodite"
        self.class_type = "Warlock"
        self.role = "Damage"
        self.route = "Middleway"
        self.resource = "Mana"
        self.health = 1300
        self.mana = 1000
        self.armor = 40
        self.magic_resistance = 35
        self.ability_power = 0
        self.love_marks = 0

    def encanto_celestial(self):
        self.love_marks += 1
        if self.love_marks == 3:
            self.love_marks = 0
            return "Explosão de energia mágica! Dano em área: 700"
        return "Marca de Desejo Ardente aplicada"

    def sopro_de_afrodite(self, target_marked=False):
        base_damage = 1150
        if target_marked:
            return f"Beijo explodiu! Dano em área reduzido: {base_damage * 0.5}"
        return f"Dano mágico causado: {base_damage}"

    def veu_do_amor(self, veil_broken=False):
        pulse_damage = 480
        final_explosion = 1300
        if veil_broken:
            return f"Véu explodiu! Dano mágico: {final_explosion}"
        return f"Pulso de dano mágico por contato: {pulse_damage}"

    def toque_de_extase(self, target_marked=False):
        base_damage = 900
        bonus_explosion = 800 if target_marked else 0
        return f"Dano contínuo: {base_damage}, Explosão bônus: {bonus_explosion}"

    def amor_eterno(self, targets=1, marked_targets=0):
        base_damage = 2500
        bonus = 0.2 * base_damage * (targets - 1)
        silence_duration = 1 if marked_targets > 0 else 0
        return f"Dano mágico total: {base_damage + bonus}, Silêncio: {silence_duration}s"

File: Wisdom-Knowledge/afrodite/afrodite.java

public class Afrodite {
    String name = "Afrodite";
    String classType = "Warlock";
    String role = "Damage";
    String route = "Middleway";
    String resource = "Mana";

    int health = 1300;
    int mana = 1000;
    int armor = 40;
    int magicResistance = 35;
    int abilityPower = 0;
    int loveMarks = 0;

    public String encantoCelestial() {
        loveMarks++;
        if (loveMarks == 3) {
            loveMarks = 0;
            return "Explosão de energia mágica! Dano em área: 700";
        }
        return "Marca de Desejo Ardente aplicada";
    }

    public String soproDeAfrodite(boolean targetMarked) {
        int baseDamage = 1150;
        if (targetMarked) {
            return "Beijo explodiu! Dano em área reduzido: " + (baseDamage * 0.5);
        }
        return "Dano mágico causado: " + baseDamage;
    }

    public String veuDoAmor(boolean veilBroken) {
        int pulseDamage = 480;
        int finalExplosion = 1300;
        if (veilBroken) {
            return "Véu explodiu! Dano mágico: " + finalExplosion;
        }
        return "Pulso de dano mágico por contato: " + pulseDamage;
    }

    public String toqueDeExtase(boolean targetMarked) {
        int baseDamage = 900;
        int bonusExplosion = targetMarked ? 800 : 0;
        return "Dano contínuo: " + baseDamage + ", Explosão bônus: " + bonusExplosion;
    }

    public String amorEterno(int targets, int markedTargets) {
        double baseDamage = 2500;
        double bonus = 0.2 * baseDamage * (targets - 1);
        int silenceDuration = markedTargets > 0 ? 1 : 0;
        return "Dano mágico total: " + (baseDamage + bonus) + ", Silêncio: " + silenceDuration + "s";
    }
}
